draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -191,6 +193,7 @@ For example, if a catalog zone lists three zones "example.com.",
 
 where `<unique-N>` is a label that tags each record in the collection.
 `<unique-N>` has an unique value in the collection.
+When different `<unique-N>` labels have the same value, the catalog zone is broken and MUST NOT be processed (see (#generalrequirements)).
```

Comment:
```suggestion
When different `<unique-N>` labels hold the same PTR (i.e. point to the same member zone), the catalog zone is broken and MUST NOT be processed (see (#generalrequirements)).
```

Reply:
@Habbie , how about this: (only added value after PTR)

```suggestion
When different `<unique-N>` labels hold the same PTR value (i.e. point to the same member zone), the catalog zone is broken and MUST NOT be processed (see (#generalrequirements)).
```